packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -127,6 +132,13 @@ export class CreateAccountStore {
         );
       }
 
+      if (this.skippedFlag) {
+        await localForage.setItem(
+          `__flagged_${parityAddress}`,
+          this.bip39Phrase || this.parityPhrase
```

Comment:
ah this is messed up actually, should encrypt the phrase with password before storing.

Reply:
maybe you can use the RPCs `parity_encryptMessage` / `parity_decryptMessage` for that